### ISSUE TITLE
Sorting Expression comparison page

### ISSUE THREAD

**Issue by fbastian** [BgeeDB/bgee_apps#220]
In GitLab by @marcrr on Sep 3, 2019, 15:19

The second column doesn't sort correctly on the Expression comparison page, e.g.

https://bgee.org/?page=expression_comparison&data=b03e80fc38f4669e6b5553fa9d42cc0212481282

**Comment by fbastian** [comment]
In GitLab by @marcrr on Sep 3, 2019, 15:21

@jwollbrett suggested bug due to treating scientific notation numbers like text, but doesn't hold: when sorting first by Min rank, sorting correct despite scientific notation, but doesn't sort by second column clicked, whatever it is.

**Comment by fbastian** [comment]
In GitLab by @jwollbrett on Sep 4, 2019, 10:02

mentioned in commit e92a95114420c9cbbb8991717488becae4978931

**Comment by fbastian** [comment]
In GitLab by @jwollbrett on Sep 4, 2019, 10:04

closed

**Comment by fbastian** [comment]
In GitLab by @jwollbrett on Sep 4, 2019, 14:46

mentioned in commit 2b59533b791c7d283dc3a18c1e7c03aaeb311b71

**Comment by fbastian** [comment]
In GitLab by @jwollbrett on Sep 4, 2019, 16:35

Sorting by one column was fine. The problem appeared when sorting using 2 or more columns.
We added 3 sorting functions. 
- one sorting the rank using scientific notation
- one sorting the column with number of genes (present, absent, no data). We added this function because these columns does not only contain the number of genes but also listed the genes. If the list of genes was different it had an impact on the sorting method even if the number of genes was equal. The new function sort using only the number of genes.
- one equivalent to the previous one but sorting for number of species (present and absent)
